http-body-util/src/collected.rs

```diff
@@ -0,0 +1,140 @@
+use std::{
+    convert::Infallible,
+    pin::Pin,
+    task::{Context, Poll},
+};
+
+use bytes::{Buf, Bytes};
+use http::HeaderMap;
+use http_body::{Body, Frame};
+
+use crate::util::BufList;
+
+/// A collected body produced by [`BodyExt::collect`] which collects all the DATA frames
+/// and trailers.
+#[derive(Debug)]
+pub struct Collected<B> {
+    pub(crate) bufs: BufList<B>,
```

Comment:
It doesn't look like these `pub(crate)` modifiers are need anymore.

Reply:
Good call removed in https://github.com/hyperium/http-body/pull/70/commits/5c5b446fe7bf002f4452bc07af816183d6b0cb65